src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,20 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
duplicate would say that the source needs also be identical, no?
Anyway, fine to change it to anything else